global.php

```diff
@@ -779,8 +779,9 @@
 
 if(!empty($moderation_queue))
 {
-	$moderation_queue = $lang->sprintf($lang->mod_notice, implode($lang->comma, $moderation_queue));
-	$moderation_queue = substr_replace($moderation_queue, ' '.$lang->and.' ', strrpos($moderation_queue, $lang->comma), strlen($lang->comma));
+	$moderation_queue_last = array_pop($moderation_queue);
+	$moderation_queue = empty($moderation_queue) ? $moderation_queue_last : implode($lang->comma, $moderation_queue).' '.$lang->and.' '.$moderation_queue_last;
```

Comment:
Please change the ternary operator to `if` to comply with [1.8 standards](https://docs.mybb.com/1.8/development/standards/).

Reply:
What about other already existing instances?
![snap](https://i.ibb.co/ypxqcYP/Untitled-1.png)

Reply:
These can be fixed later too for readability (not that important as new code in 1.9 will use PSR standards).

Reply:
Ternary operator removed in 6ec0619.